engine/lib/metastrings.php

```diff
@@ -27,6 +27,18 @@ function elgg_get_metastring_id($string, $case_sensitive = true) {
 }
 
 /**
+ * Get a map of metastring identifiers to string values (case sensitively)
```

Comment:
Isn't it the other way around?

Might be also good to mention in the docblock what use cases the function in meant for.